tests/backwards_compatibility_tests/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest
```

Comment:
we should pin the versions for the deps

Reply:
[done](https://github.com/marqo-ai/marqo/pull/1024/commits/740097f76bc5fd67568d409c1a680d9ebcb77333#diff-25462d20a86fa4940aa7672256f7c22ee5184a4b439fe7a74a2527aed0e673c8R1-R6)